### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.4 - es. 3.30 (14 ottobre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#292] [Esercizi M-Z]
![es.30](https://github.com/user-attachments/assets/33c664e8-3e7c-4d67-861e-7bc38b848c03)

**Comment by marcoseverini** [comment]
<img width="687" alt="immagine" src="https://github.com/user-attachments/assets/25a930e3-9eb0-4760-b000-ca28ee1b1f57">

**Reply by alem1105** [reply]
Nella seconda domanda invece di usare $P(E)$ (probabilita' del sasso nero) non va usato $P(E^C)$ dato che sappiamo che abbiamo estratto un sasso bianco?

**Reply by marcoseverini** [reply]
Grazie per averlo specificato, ho fatto un errore di distrazione

**Comment by alem1105** [comment]
<img width="1023" alt="375966054-51fa2817-6982-490e-b15a-5f86a1ec6b86" src="https://github.com/user-attachments/assets/5583b191-641e-4ae9-ad68-e3cdae6c743e">
